src/skin/tooltips.cpp

```diff
@@ -637,4 +652,102 @@ void Tooltips::addStandardTooltips() {
     add("starrating")
             << tr("Star Rating")
             << tr("Assign ratings to individual tracks by clicking the stars.");
+
+    // Effect Unit Controls
+    add("EffectUnit_clear")
+            << tr("Clear Unit")
+            << tr("Clear effect unit.");
+
+    add("EffectUnit_enabled")
+            << tr("Toggle Unit")
+            << tr("Toggle effect unit.");
```

Comment:
"Enable or disable effect processing."

We should file a bug that our effects cut out instantly when this is done, whereas most people expect effects like reverb to linger after they are disabled.  (Post-fader vs prefader I think)